docs/rancher/resource-quota.md

```diff
@@ -0,0 +1,56 @@
+
+ResourceQuota can define quota limits for the following resources:
+
+- CPU (compute resources): Limits the usage of compute resources in terms of CPU cores or CPU time.
+- Memory (memory resources): Limits the usage of memory resources in bytes or other recognizable memory units.
+Using `ResourceQuota`, administrators can set different resource limits for each namespace, ensuring the proper utilization and allocation of resources. When a namespace reaches its resource quota limit, further resource requests are denied, preventing excessive consumption of cluster resources and ensuring the normal operation of other namespaces.
+
+In summary, `ResourceQuota` is a mechanism in Kubernetes used to restrict and manage the usage of resources within namespaces, promoting fair resource sharing and proper allocation.
+
+## ResourceQuota in Harvester
+In multi-cluster management mode, Harvester supports administrators in setting resource quotas for namespaces, as follows:
+
+- Click `⋮` -> `Virtualization Management` -> `Import Harvester Cluster` -> `Project/Namespace`.
+- Click `Create Project`.
+- Enter the project name, click `Resource Quotas` and Add Resource Type, select `CPU Limit` or `Memory Limit` and enter the quota value, and click `Create`.
+  ![](/img/v1.2/rancher/create-project.png)
+- Under the newly created Project, click `Create Namespace`.
+- Enter the Namespace name, which supports adjusting for the default quota, and click `Create`.
+  ![](/img/v1.2/rancher/create-namespace.png)
+
+## Overhead Memory of Virtual Machine
+When creating a virtual machine, the virtual machine controller will automatically add overhead resources to the virtual machine. The overhead resources are used to ensure the regular operation of the virtual machine, and the configuration of memory limits requires consideration of reserving more memory.
+
+For more information, see the [Memory Overhead documentation](https://kubevirt.io/user-guide/virtual_machines/virtual_hardware/#memory-overhead).
+
+For more information on how to calculate memory overhead, refer to [Code](https://github.com/kubevirt/kubevirt/blob/v0.54.0/pkg/virt-controller/services/template.go#L1804).
+
+
+
+## Adjust ResourceQuota automatically during migration
+When the resource quota defined by the `ResourceQuota` object is exhausted, virtual machine migration is not supported. Migration requires creating a pod with the same resources as the migrated VM. If the resources required for creating the pod exceed the quota, migration cannot proceed.
+
+In Harvester, before migration, the resource limits of `ResourceQuota` are increased based on the specifications of the target virtual machine. After the migration, the resource limits are restored (reduced) to their previous values.
```

Comment:
```suggestion
In Harvester, before the migration, the values of `ResourceQuota` are automatically increased based on the resources required by the target virtual machine. After the migration, `ResourceQuotas` are restored (reduced) to their previous values.
```

Reply:
`resource limit` is actually the same as `ResourceQuota`? if so, prefer using a consistent term